packages/sirius-web/frontend/sirius-web-application/src/modals/new-document/NewDocumentModal.tsx

```diff
@@ -158,6 +168,8 @@ export const NewDocumentModal = ({ editingContextId, onClose }: NewDocumentModal
           const { message } = createDocument;
           const showToastEvent: ShowToastEvent = { type: 'SHOW_TOAST', message };
           dispatch(showToastEvent);
+        } else if (isSuccessPayload(createDocument)) {
+          setSelection({ entries: [{ id: createDocument.document.id, label: 'New', kind: 'siriusWeb://document' }] });
```

Comment:
Why invent a name here instead of using the one from the GraphQL API? We could also retrieve the kind in case this changes later.